tools/slurm/scripts/slurmcluster.sh

```diff
@@ -146,11 +169,11 @@ echo "Started bidirectional tunnels to $INSTANCE_NAME"
 # Grab launcher token.
 REMOTE_TOKEN_SOURCE=/opt/launcher/jetty/base/etc/.launcher.token
 LOCAL_TOKEN_DEST=$TEMPDIR/.launcher.token
-gcloud compute scp --quiet --zone "us-west1-b" --project "determined-ai" root@$INSTANCE_NAME:$REMOTE_TOKEN_SOURCE $LOCAL_TOKEN_DEST
+tryloop 1 10 30 -- gcloud compute scp --quiet --zone "us-west1-b" --project "determined-ai" root@$INSTANCE_NAME:$REMOTE_TOKEN_SOURCE $LOCAL_TOKEN_DEST
 echo "Copied launcher token to $LOCAL_TOKEN_DEST"
 # The launcher service verifies communication with the launcher, so just have to be sure it is started.
 # Also show the status for extra confirmaiton in the logs of the state.
-gcloud compute ssh --zone "$ZONE" "$INSTANCE_NAME" --project "$PROJECT" -- "sudo systemctl start launcher.service ; systemctl status launcher.service --no-pager"
+tryloop 1 10 30 -- gcloud compute ssh --zone "$ZONE" "$INSTANCE_NAME" --project "$PROJECT" -- "sudo systemctl start launcher.service ; systemctl status launcher.service --no-pager"
```

Comment:
Maybe we want to try 1 a few times before waiting so long. I'm cool either way, though.

Reply:
I can do that